frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -409,7 +409,7 @@ type MemoFieldRowProps = {
   showMyInfoBadge?: boolean
 }
 
-const MemoFieldRow = memo(({ field, ...rest }: MemoFieldRowProps) => {
+const MemoFieldRow = ({ field, ...rest }: MemoFieldRowProps) => {
```

Comment:
should we rename this if we are not memoizing this code?

Reply:
done